km_api/know_me/tests/serializers/test_emergency_contact_serializer.py

```diff
@@ -0,0 +1,93 @@
+from know_me import serializers
+
+
+def test_create(km_user_factory):
+    """
+    Saving a serializer containing valid data should create a new
+    emergency contact attached to the given KMUser.
+    """
+    km_user = km_user_factory()
+    data = {
+        'name': 'My Name',
+        'relation': 'My relation',
+        'phone_number': '19199911919',
+        'alt_phone_number': '18008008000',
+        'email': 'email@gmail.com'
+    }
+
+    serializer = serializers.EmergencyContactSerializer(data=data)
+    assert serializer.is_valid()
+
+    emergencycontact = serializer.save(km_user=km_user)
+
+    assert emergencycontact.name == data['name']
+
+
+def test_has_object_read_permission_owner(api_rf, emergency_contact_factory):
```

Comment:
The tests for object permissions should be in the `test_emergency_contact_model.py` file.